File: Examples/CocoapodsExample/ObjCExample/GeofenceObjCPodsExample/ViewController.m
//
//  ViewController.m
//  GeofenceObjCPodsExample
//
//  Created by Yogesh Singh on 09/07/20.
//  Copyright © 2020 CleverTap. All rights reserved.
//

#import "ViewController.h"
@import CoreLocation;

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

- (IBAction)requestLocationPermission:(UIButton *)sender {
    CLLocationManager *locationManager = [[CLLocationManager alloc] init];
    [locationManager requestAlwaysAuthorization];
}

@end

File: Examples/CarthageExample/GeofenceCarthageExample/ViewController.swift
//
//  ViewController.swift
//  GeofenceCarthageExample
//
//  Created by Yogesh Singh on 02/09/20.
//  Copyright © 2020 CleverTap. All rights reserved.
//

import UIKit
import CoreLocation

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }

    @IBAction func requestLocationPermission(_ sender: UIButton) {
        let locationManager = CLLocationManager()
        locationManager.requestAlwaysAuthorization()
    }
}

